doc/source/getting_started/index.rst

```diff
@@ -10,6 +10,30 @@ generator for creating documentation. If you are new to using Sphinx, see
 This section explains how to install the Ansys Sphinx theme and then set up your
 Sphinx ``conf.py`` file to use this theme to generate your documentation.
 
+Dependencies
+------------
+
+Ansys sphinx theme build on top of ``pydata sphinx theme``. 
+The theme requires the following dependencies:
+
+- `sphinx <https://www.sphinx-doc.org/en/master/>`_
+- `pydata-sphinx-theme <https://pydata-sphinx-theme.readthedocs.io/en/latest/>`_
+- `Jinja2 <https://jinja.palletsprojects.com/en/2.11.x/>`_
```

Comment:
Since we are talking about dependencies here, I guess we could point to the official pages in PyPI. What do you think?

Reply:
yeah sure, i will add those.